qa/sc_cert_submitter_secure_enclave.py

```diff
@@ -93,20 +85,20 @@ def run_test(self):
         mc_node.generate(mc_blocks_left_for_we + 1)
 
         # Generate 2 SC blocks on SC node and start them automatic cert creation.
-        generate_next_block(sc_node1, "second node")  # 1 MC block to reach the end of WE
-        generate_next_block(sc_node1, "second node")  # 1 MC block to trigger Submitter logic
+        generate_next_block(sc_node1, "first node")  # 1 MC block to reach the end of WE
+        generate_next_block(sc_node1, "first node")  # 1 MC block to trigger Submitter logic
         self.sc_sync_all()  # Sync SC nodes
 
         # Wait for Certificates appearance
-        time.sleep(10)
-        while mc_node.getmempoolinfo()["size"] < 1 and sc_node2.submitter_isCertGenerationActive()["result"]["state"]:
+        time.sleep(100)
+        while mc_node.getmempoolinfo()["size"] < 1 and sc_node1.submitter_isCertGenerationActive()["result"]["state"]:
             logging.info("Wait for certificates in the MC mempool...")
-            if sc_node2.submitter_isCertGenerationActive()["result"]["state"]:
-                logging.info("sc_node2 generating certificate now.")
+            if sc_node1.submitter_isCertGenerationActive()["result"]["state"]:
+                logging.info("sc_node1 generating certificate now.")
             time.sleep(2)
         assert_equal(1, mc_node.getmempoolinfo()["size"], "Certificates was not added to MC node mempool.")
-        logging.info("Node after synchronization was able to sign, collect signatures and emit certificate.")
-        api_server.stop()
+        logging.info("Node with Secure Enclave was able to sign, collect signatures and emit certificate.")
```

Comment:
let's check that cert in mempool has quality = 6, to be sure that both 3 keys from wallet and 3 keys from enclave were used.